### ISSUE TITLE
Unable to get composite pid

### ISSUE THREAD

**Issue by Weab-chan** [SnowNF/wra#1]
Hi, 

Would it be possible to add a bit more detail in your readme? Is the termux script meant to be run under root?

Thanks

**Comment by Weab-chan** [comment]
Solved by adding sudo in front of the command.. same thing for the kill command.. now im getting a black screen

**Comment by SnowNF** [comment]
Yes.. Actually this script just stop the android compositor and start the labwc without handle any exceptions. It's highly recommend to follow [this article](https://zhuanlan.zhihu.com/p/687261201) and then use this script with suitable modification
